src/projects/version.md

```diff
@@ -0,0 +1,7 @@
+# Project Versioning
+
+Radare2 have its own versioning control system. It's like a simple `git`. But also acts as a way to wrap the use of **git** under the same API.
```

Comment:
The first sentence should justify/state WHY there's such a custom system and not just use plain `git`? Or at least compare with the state of the art in other communities/programs? For example:

If (I guess) the message you want to convey here is that you want to have r2 talk to git via a simplified wrapper, you can bring up [`pass`](https://www.passwordstore.org/), which does something similar: `pass git push`, `pass git pull`, etc... and it rarely hits conflicts with the simplified wrapper, which is an advantage to users vs having a full fledged (unwrapped) git underneath.